tests/late_nodes/delayed_config.yaml

```diff
@@ -41,7 +41,7 @@ bootstrap:
 client:
   image: 'spacemeshos/go-spacemesh:develop'
   command: '/bin/go-harness'
-  replicas: 49
+  replicas: 20
```

Comment:
why?

Reply:
this is the value from the original test